### PR TITLE
Update E-core TMA 3.60 CSV/XLSX

### PR DESCRIPTION
**Pull request by rpmclaug**

### CODE REVIEW

**Review by claudiaromo — APPROVED** [review]

**Comment by rpmclaug** [comment]
* While adopting the DurationTimeInMilliseconds formula, it triggers the inheritance search using ratio_column for GRT. ('GRT') was originally downcast to a string instead of a tuple of one element, resulting in walking over characters. Fix: Add comma in ratio_column for GRT to force it to be a tuple of 1 element.
* DurationTimeInMilliseconds appears to be effectively dropped because it's a knob with no value. Automation appears to use the intermediate Aux metric #DurationTimeInSeconds. Fix: added this intermediate metric as well.
* Added CMT entry to ratio_column as well, as searching ratio_column was failing.

File: scripts/create_perf_json.py
Comment (weilinwa) [inline scripts/create_perf_json.py:1]: Hi @rpmclaug, just a FYI, I added some changes to this script for TMA 4.8 and E-Core TMA 3.6 updates. I will figure out a way to merge that into the repo.

Comment (rpmclaug) [reply scripts/create_perf_json.py:1]: Thanks @weilinwa , what's the best way to proceed? Should we merge this PR after approval, or should I wait and rebase onto your upcoming merge?

Comment (weilinwa) [reply scripts/create_perf_json.py:1]: @rpmclaug, for TMA4.8, we created a separate PR after the major TMA4.8 release PR for perf JSON. If we want to do similar thing here, I think you could merge this PR first. I will generate perf JSON use the converter script and submit another PR after that. 

My changes to the script will be pushed in later (after the perf JSON file PR get done). It resolves these issues and also adds some other changes. There might be some git merge conflict at that time, but we should be able to resolve that easily.

**Comment by rpmclaug** [comment]
@captain5050 If you have the time, I'd appreciate your review before we merge any changes to the create_perf_json script.

**Review by edwarddavidbaker — APPROVED** [review]
LGTM. Thanks Ryan.

----

The trailing GRT comma looked familiar to me. I introduced the issue in an earlier review https://github.com/intel/perfmon/pull/145#discussion_r1504549537 ! Thanks for fixing :).

**Pull request merged**
